Android/app/build.gradle

```diff
@@ -36,7 +36,7 @@ dependencies {
     compile fileTree(include: ['*.jar'], dir: 'libs')
     testCompile 'junit:junit:4.12'
 
-    compile 'com.squareup.okhttp3:okhttp:3.4.1'
+    compile 'com.squareup.okhttp3:okhttp:3.9.1'
```

Comment:
Is there a need to upgrade the version?

Reply:
Not necessary to update it. I thought it'll be better if we keep 3rd party libraries up-to-date.